cumulus/pallets/ah-ops/src/lib.rs

```diff
@@ -260,6 +293,40 @@ pub mod pallet {
 
 			Self::do_unreserve_crowdloan_reserve(block, depositor, para_id).map_err(Into::into)
 		}
+
+		/// Try to migrate a parachain sovereign child account to its respective sibling.
+		///
+		/// Takes the old and new account and migrates it only if they are as expected. An event of
+		/// `SovereignMigrated` will be emitted if the account was migrated successfully.
+		///
+		/// Callable by any signed origin.
+		#[pallet::call_index(3)]
+		#[pallet::weight(T::DbWeight::get().reads_writes(15, 15)
+					.saturating_add(Weight::from_parts(0, 50_000)))]
+		pub fn migrate_parachain_sovereign_acc(
+			origin: OriginFor<T>,
+			from: T::AccountId,
+			to: T::AccountId,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			Self::do_migrate_parachain_sovereign_acc(&from, &to).map_err(Into::into)
+		}
+
+		/// Force unreserve a named or unnamed reserve.
+		#[pallet::call_index(4)]
+		#[pallet::weight(T::DbWeight::get().reads_writes(15, 15)
```

Comment:
probably less

Reply:
Put it to 10, probably still paranoid but better than underestimate 😆